Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -608,6 +608,19 @@ public void OnMouseEvent (MouseEventArgs a)
     /// <param name="ctrl">If <see langword="true"/> simulates the Ctrl key being pressed.</param>
     public abstract void SendKeys (char keyChar, ConsoleKey key, bool shift, bool alt, bool ctrl);
 
+    /// <summary>
+    ///     Provide handling for the terminal write ANSI escape sequence request.
+    /// </summary>
+    /// <param name="ansiRequest">The <see cref="AnsiEscapeSequenceRequest"/> object.</param>
+    /// <returns>The request response.</returns>
+    public abstract string WriteAnsiRequest (AnsiEscapeSequenceRequest ansiRequest);
+
+    /// <summary>
+    ///     Provide proper writing to send escape sequence recognized by the <see cref="ConsoleDriver"/>.
+    /// </summary>
+    /// <param name="ansi"></param>
+    public abstract void WriteRaw (string ansi);
```

Comment:
That why I made it abstract because depends of the driver. Only the `NetDriver` don't implement because the `EscSeqRequests.Add` method call the `Console.Out.Write` if the driver parameter is null.